Source/DataSources/CzmlDataSource.js

```diff
@@ -1973,6 +1995,7 @@ define([
      * @param {String|Object} czml A url or CZML object to be processed.
      * @param {Object} [options] An object with the following properties:
      * @param {String} [options.sourceUri] Overrides the url to use for resolving relative links.
+     * @param {Object} [options.query] Key-value pairs which are appended to all URIs in the CZML.
```

Comment:
`CzmlDataSource.prototype.load` and `CzmlDataSource.prototype.process` most likely need similar updates to their documentation.